python/lsst/jointcal/jointcal.py

```diff
@@ -77,6 +90,7 @@ class JointcalConfig(pexConfig.Config):
         doc = "How to select sources for cross-matching",
         default = "astrometry"
     )
+    # TODO: CmdLineTask has a profile thing built-in: can we tie in to that?
```

Comment:
I agree, but how does one manage turning on/off profiling in general? I want to be able to enable profiling when timing various things, but I don't want it on by default. Is there a standard approach to that?